suite-native/device/src/hooks/useHandleDeviceConnection.ts

```diff
@@ -53,14 +53,6 @@ export const useHandleDeviceConnection = () => {
     // set connecting screen to be displayed again on the next device connection.
     useEffect(() => {
         if (isNoPhysicalDeviceConnected && isOnboardingFinished) {
-            // This accidentally gets triggered by finishing onboarding with no device connected,
```

Comment:
This is not useful anymore? I think it's still nice not to redirect twice

Reply:
I can put it back. Thought it would be simpler like this and doesn't seem to affect performance (haven't measured this)